paper-input-container.html

```diff
@@ -83,6 +83,7 @@
 `--paper-input-container-color` | Label and underline color when the input is not focused | `--secondary-text-color`
 `--paper-input-container-focus-color` | Label and underline color when the input is focused | `--primary-color`
 `--paper-input-container-invalid-color` | Label and underline color when the input is is invalid | `--error-color`
+`--paper-input-container-required-color` | Color used for the asterix displayed for required input | `--error-color`
```

Comment:
I am not sure there's a need for this. From the mocks, the `*` is the same color as the error color, and I don't think there's a need for that to be different.